clang/unittests/Format/FormatTest.cpp

```diff
@@ -11077,6 +11077,221 @@ TEST_F(FormatTest, WrapsTemplateDeclarationsWithComments) {
       Style);
 }
 
+TEST_F(FormatTest, BreakBeforeTemplateClose) {
```

Comment:
One additional test: How about nested templates?
`template<template<typename, typename> typename T>` for example.

Reply:
Sure! https://github.com/llvm/llvm-project/pull/118046/commits/2c3a64c8f32a5c43887db4d107a4142490898c20

Reply:
This one is still missing.

Reply:
Sorry if I misunderstood, but, I thought I added that here: 
![Screenshot_20241224-154618](https://github.com/user-attachments/assets/ce0a3af9-421e-48a9-a8ec-9b443f26b324)

Reply:
Sorry missed that, didn't look for the lambda case.